open_discussions/settings.py

```diff
@@ -123,7 +123,6 @@
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
     "corsheaders.middleware.CorsMiddleware",
     "authentication.middleware.BlockedIPMiddleware",
-    "open_discussions.middleware.user_activity.UserActivityMiddleware",
```

Comment:
This was added by @rhysyngsun for https://github.com/mitodl/open-discussions/issues/473 - is it not needed anymore?

Reply:
Yeah, we can just remove that entirely.